samples/powerpoint/scenarios/searches-wikipedia-api.yaml

```diff
@@ -0,0 +1,233 @@
+order: 1
+id: powerpoint-scenarios-searches-wikipedia-api
+name: Searches Wikipedia
+description: Searches Wikipedia based on the selected text in the presentation.
+author: cakriwut
+host: POWERPOINT
+api_set: {}
+script:
+    content: |
+        declare let moment: any;
```

Comment:
@cakriwut thanks for the updates! Would you mind also adding your copyright here at the top of the sample? We like to give credit for every contribution! The typical format would be:
/*
 * Copyright (c) Riwut Libinuko. All rights reserved. Licensed under the MIT license.
 */
or similar...
Thanks!

Reply:
Thanks for the acceptance.